docs/installation/docker.md

```diff
@@ -0,0 +1,111 @@
+--
+layout: doc
+title: Running with Docker
+order: 8
+--
+
+# Running with Docker
+
+Dovecot CE comes with Docker images published at https://hub.docker.com/r/dovecot/dovecot.
+
+## Minimal setup
+
+To run Dovecot you can start it with 
+
+```
```

Comment:
For command line examples, adding "console" to block markdown will properly highlight.  i.e.: "```console"